chromadb/server/fastapi/__init__.py

```diff
@@ -195,6 +196,134 @@ def __init__(self, settings: Settings):
+            response_model=None,
+        )
+        self.router.add_api_route(
+            "/api/v2/tenants/{tenant}/databases/{database}/collections_count",
+            self.count_collections,
+            methods=["GET"],
+            response_model=None,
+        )
+        self.router.add_api_route(
+            "/api/v2/tenants/{tenant}/databases/{database}/collections",
+            self.create_collection,
+            methods=["POST"],
+            response_model=None,
+        )
+
+        self.router.add_api_route(
+            "/api/v2/tenants/{tenant}/databases/{database}/collections/{collection_id}/add",
+            self.add,
+            methods=["POST"],
+            status_code=status.HTTP_201_CREATED,
+            response_model=None,
+        )
+        self.router.add_api_route(
+            "/api/v2/tenants/{tenant}/databases/{database}/collections/{collection_id}/update",
+            self.update,
+            methods=["POST"],
+            response_model=None,
+        )
+        self.router.add_api_route(
+            "/api/v2/tenants/{tenant}/databases/{database}/collections/{collection_id}/upsert",
+            self.upsert,
+            methods=["POST"],
+            response_model=None,
+        )
+        self.router.add_api_route(
+            "/api/v2/tenants/{tenant}/databases/{database}/collections/{collection_id}/get",
+            self.get,
+            methods=["POST"],
+            response_model=None,
+        )
+        self.router.add_api_route(
+            "/api/v2/tenants/{tenant}/databases/{database}/collections/{collection_id}/delete",
+            self.delete,
+            methods=["POST"],
+            response_model=None,
+        )
+        self.router.add_api_route(
+            "/api/v2/tenants/{tenant}/databases/{database}/collections/{collection_id}/count",
+            self.count,
+            methods=["GET"],
+            response_model=None,
+        )
+        self.router.add_api_route(
+            "/api/v2/tenants/{tenant}/databases/{database}/collections/{collection_id}/query",
+            self.get_nearest_neighbors,
+            methods=["POST"],
+            response_model=None,
+        )
+        self.router.add_api_route(
+            "/api/v2/tenants/{tenant}/databases/{database}/collections/{collection_name}",
+            self.get_collection,
+            methods=["GET"],
+            response_model=None,
+        )
+        self.router.add_api_route(
+            "/api/v2/tenants/{tenant}/databases/{database}/collections/{collection_id}",
+            self.update_collection,
+            methods=["PUT"],
+            response_model=None,
+        )
+        self.router.add_api_route(
+            "/api/v2/tenants/{tenant}/databases/{database}/collections/{collection_name}",
+            self.delete_collection,
+            methods=["DELETE"],
+            response_model=None,
+        )
+
+        # ======================================================================
```

Comment:
Nit could we move the root bootstrapping into functions per version to make this less of a wall of text?